lib/matplotlib/axes/_axes.py

```diff
@@ -1962,6 +1961,10 @@ def acorr(self, x, **kwargs):
             The marker for plotting the data points.
             Only used if *usevlines* is ``False``.
 
+        **kwargs
+            Additional properties passed to `.Axes.vlines` and `.Axes.axhline`
```

Comment:
```suggestion
            Additional parameters are passed to `.Axes.vlines` and
            `.Axes.axhline` if *usevlines* is ``True``; otherwise they are
            passed to `.Axes.plot`.
```

Also for `xcorr`.